ci/long_running_tests/workloads/serve_failure.py

```diff
@@ -23,12 +23,12 @@
         resources={str(i): 2},
         object_store_memory=object_store_memory,
         redis_max_memory=redis_max_memory,
-        webui_host="0.0.0.0")
+        dashboard_host="0.0.0.0")
 
 ray.init(
     address=cluster.address,
-    include_webui=True,
-    webui_host="0.0.0.0",
+    include_dashboard=True,
```

Comment:
Don't need include_dashboard right?

Reply:
Yeah, you're correct actually. I mindlessly put it there because we had passed include_webui before hehe.